snapcraft.yaml

```diff
@@ -0,0 +1,28 @@
+name: oras
+version: git
+summary: Package for the ORAS CLI tool
+description: |
+  A snap package for the ORAS CLI tool, which is used for managing OCI artifacts.
+base: core22
+confinement: strict
+grade: devel
+
+parts:
+  oras:
+    plugin: go
+    source: .
+    build-snaps:
+      - go/1.20/stable
+    build-packages:
+      - make
+      - ca-certificates
```

Comment:
I don't know how snapcraft works, what is this target `ca-certifacates`?

Reply:
My base for this snap was this dockerfile: https://github.com/oras-project/oras/blob/main/Dockerfile so I used this package because you used it in the dockerfile .

Reply:
and you need it to run oras I assume

Reply:
Ah, I thought that might have been `make ca-certificates`, but it is make and ca-certificates. My confusion

Reply:
np thanks to you, i saw that ca-certificates needed to be in stage-package not in build-packages